server/controllers/notifications.js

```diff
@@ -0,0 +1,107 @@
+const Notification = require("../models/Notifications");
+const asyncHandler = require("express-async-handler");
+const mongoose = require("mongoose");
+const { ObjectId } = mongoose.Types;
+
+// @route POST /notification
+// @desc create a new notification
+// @access Private
+exports.createNotification = asyncHandler(async (req, res, next) => {
+  try {
+    const userId = req.user.id;
+
+    const { type, title, description } = req.body;
```

Comment:
Make sure to validate all these fields and return a bad request status if any is missing.

Reply:
done